cdap-runtime-spi/pom.xml

```diff
@@ -39,6 +39,11 @@
       <groupId>org.apache.twill</groupId>
       <artifactId>twill-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.cdap.cdap</groupId>
+      <artifactId>cdap-watchdog-api</artifactId>
```

Comment:
Better not to pull in this dependency, as it depends on cdap-api. The problem with dependency on cdap-api is that it pulls in old gson, which could affect provisioner dependencies.

Reply:
fixed